src/components/Board.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import Square from './Square';
+import BoardContext from '../context/BoardContext'
+
+function Board(props) {
+    const { board, setBoard, turn, setTurn, history, setHistory } = useContext(BoardContext);
+
+    function declareWinner(board) {
+        const winningCombos = [
+            [3, 4, 5],
+            [0, 1, 2],
+            [0, 3, 6],
+            [2, 5, 8],
+            [1, 4, 7],
+            [0, 4, 8],
+            [2, 4, 6],
+            [6, 7, 8],
+        ];
+
+        function threeInARow() {
+            for (let i = 0; i < winningCombos.length; i++) {
+                const [first, second, third] = winningCombos[i];
+                // When you get 3 in a row, in any direction.
+                if (board[first] && board[first] === board[second] && board[first] === board[third]) {
+                    return board[first];
+                }
+            }
+            return null;
+        }
+        threeInARow()
```

Comment:
I think you need a `return` statement here to either return `null` or the winner.